packages/mdc-chips/chip/index.js

```diff
@@ -47,6 +47,22 @@ class MDCChip extends MDCComponent {
     return new MDCChip(root);
   }
 
+  initialize() {
+    this.leadingIcon_ = this.root_.querySelector(strings.LEADING_ICON_SELECTOR);
+    this.ripple_ = new MDCRipple(this.root_);
+
+    // Adjust ripple size for chips with animated growing width. This applies when filter chips without
+    // a leading icon are selected, and a leading checkmark will cause the chip width to expand.
+    const checkmarkEl = this.root_.querySelector(strings.CHECKMARK_SELECTOR);
+    if (!!checkmarkEl && !this.leadingIcon_) {
+      const height = this.root_.getBoundingClientRect().height;
+      // The checkmark width is initially set to 0, so use the chip's height as a proxy since the
```

Comment:
If only the checkmark's width is set to 0, shouldn't this use the checkmark's height rather than the chip's? Wouldn't the chip be taller than the checkmark?